flink-read-write-hive/README.md

```diff
@@ -57,21 +56,28 @@ to run this example.
    $ docker exec -w /root/flink-read-write-hive flink bash -c "export HADOOP_CLASSPATH=\`hadoop classpath\`; python main.py"
    ```
 
-   You should be able to find the following information printed out in the end
-   of the terminal.
+   The program would create and submit two Flink jobs, the second of which runs
+   infinitely as the data stream read from Hive will not be regarded as finished.
+   You may open a second terminal and execute the following command to check the
+   results saved in local file.
+
+   ```
+   cat data/output/.part-*
+   ```
+
+   You should be able to find the following information printed out in the
+   terminal.
 
    ```
-     item_id  price                  timestamp
-   0  item_1  100.0  2022-01-01 00:00:00 +0800
-   1  item_2  200.0  2022-01-01 00:00:00 +0800
-   2  item_3  300.0  2022-01-01 00:00:00 +0800
-   3  item_1  200.0  2022-01-01 00:01:30 +0800
-   4  item_1  300.0  2022-01-01 00:02:30 +0800
-   5  item_1  400.0  2022-01-01 00:03:30 +0800
+   user_1,item_1,1,"2022-01-01 00:00:00",100.0,100.0
+   user_1,item_2,2,"2022-01-01 00:01:00",200.0,500.0
+   user_1,item_1,3,"2022-01-01 00:02:00",200.0,1100.0
+   user_2,item_1,1,"2022-01-01 00:03:00",300.0,300.0
+   user_1,item_3,2,"2022-01-01 00:04:00",300.0,1200.0
    ```
 
-5. Tear down the Flink cluster and Hive cluster after the FeatHub program has
-   finished.
+5. Press Control-C to stop the infinitely-running program and tear down the Flink
```

Comment:
It is uncommon to say a Flink streaming job is "infinitely-running". Maybe remove it for simplicity?